client/src/components/InputsModal/InputsModal.tsx

```diff
@@ -84,14 +88,18 @@ const InputsModal: FC<InputsModalProps> = ({
         );
       default:
         return (
-          <ListItem key={`requirement${index}`}>
+          <ListItem key={`requirement${index}`} disabled={requirement.locked}>
             <TextField
+              disabled={requirement.locked}
               id={`requirement${index}_input`}
               className={styles.inputField}
               fullWidth
               label={requirement.title || requirement.name}
               helperText={requirement.description}
-              value={inputsMap.get(requirement.name)}
+              value={
+                (inputsMap.get(requirement.name) ?? '') +
+                (requirement.locked ? ' (*locked input)' : '')
+              }

```

Comment:
Would it be possible to have this indicator on the label of the input, instead of within the value itself?  Or perhaps setting a class on the input label, and we can adjust it with CSS?  I think that it is a bit confusing when the value of a field is populated with stuff that isn't part of the value.

I'm not sure how much sense it makes to do something like this, but in Inferno Program Edition we decided to put all "locked" inputs into a separate section to avoid confusion about what the user should enter.  
![Screen Shot 2021-09-14 at 3 50 20 PM](https://user-images.githubusercontent.com/412901/133324471-ae6004c8-29bf-486b-bc79-9931b5b50631.png)

Reply:
I really like the look of this separate section for 'locked inputs' and am going to head down that route.